components/hab/src/command/studio/enter.rs

```diff
@@ -13,12 +13,39 @@ use crate::{config,
 
 pub const ARTIFACT_PATH_ENVVAR: &str = "ARTIFACT_PATH";
 
+const AUTH_TOKEN_ENVVAR: &str = "HAB_AUTH_TOKEN";
```

Comment:
This is defined in [`components/core/src/lib.rs`](https://github.com/habitat-sh/habitat/blob/master/components/core/src/lib.rs#L28), how do I use that value instead of defining it here?  (e.g.: if I comment this line out, I get a "not found in this scope" error)

Reply:
The compiler error gives good advice here:
```
error[E0425]: cannot find value `AUTH_TOKEN_ENVVAR` in this scope
  --> components/hab/src/command/studio/enter.rs:36:29
   |
36 |     set_env_var_from_config(AUTH_TOKEN_ENVVAR, config.auth_token);
   |                             ^^^^^^^^^^^^^^^^^ not found in this scope
help: possible candidate is found in another module, you can import it into scope
   |
1  | use habitat_core::AUTH_TOKEN_ENVVAR;
```
Adding
```rust
use habitat_core::AUTH_TOKEN_ENVVAR;
```
is what you want.

Reply:
doh.  I even saw that...